### ISSUE TITLE
Set the current page on the top menu

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#5199]
Our accessibility audit recommended to include a visual status on the top menu for the current page.
Currently it is not visible from the menu which page or section the user is viewing.

On that note, it is recommended to enable on the menu the `aria-current="page"` on link item menu for screenreaders being able to announce the current page link to the user.

**Comment by israelcefrin** [comment]
@Vitaliy-1 are you currently working on some issue related to this request for marking the current page on top menu?

**Comment by israelcefrin** [comment]
Hi @jardakotesovec ,
Just bringing your question from https://github.com/pkp/ojs/pull/4133 to this issue:
> should not we also update the other cases to be links? For example if we go to /index.php/publicknowledge/about, than About the Journal is not link.

Since the " about " element is not a link to anywhere but a menu opener, it should be changed from `<a href..` to `<button>` element. We can even open an issue just for this improvement since it will require HTML/CSS and JS tweaking. 

See the [W3C website menu ](https://www.w3.org/). It opens, closes, showing an icon change and highlight color to show when active.

**Comment by israelcefrin** [comment]
OMP 3.4 PR - https://github.com/pkp/omp/pull/1505
